pages/dashboard/index.tsx

```diff
@@ -21,10 +28,72 @@ const DashboardPage: FC<DashboardPageProps> = observer(({ jwtPayload }) => {
 
   return (
     <SessionBox title={t('backend_management')} path={asPath} {...{ menu, jwtPayload }}>
-      <Container maxWidth="lg" className="py-16 text-center">
-        <Typography variant="h3" component="h1">
+      <Container maxWidth="lg" className="py-8">
+        <Typography variant="h3" component="h1" gutterBottom>
           {t('welcome_use')}
         </Typography>
+
+        {/* New Project Section */}
+        <form
+          style={{ marginTop: 32, marginBottom: 32 }}
+          onSubmit={event => {
+            event.preventDefault();
+
+            const { title } = formToJSON<{ title: string }>(event.currentTarget);
+
+            location.href = `/dashboard/project/new?title=${encodeURIComponent(title)}`;
```

Comment:
```suggestion
          onSubmit={async event => {
            event.preventDefault();

            const { title } = formToJSON<{ title: string }>(event.currentTarget);

            const { id } = await projectStore.updateOne({ title });

            location.href = `/dashboard/project/${id}`;
```
把这个回调放到 `/pages/requirement.tsx` 吧，公益版、商业版统一输入框，只是点不同提交按钮近不同的页面，本页面就无需创建表单了。

同时在这个统一输入框下方做两个并排卡片说明版本区别：
- 公益版：
    1. GitHub 账号一键免注册登录
    2. 免费评估，每日限额
    3. 提交的资料全网公开
    4. [开源市集](https://bazaar.fcc-cd.dev/)等开源社区志愿者不定期回复、跟进
    5.  项目开发最终由上述志愿者以[开源悬赏](https://idea2app.feishu.cn/wiki/I348wiT7DipbKbkPUPfcwQBAn7c)的方式落地
- 商业版：
    1. 手机号一键免密注册登录
    2. 全天候不限量评估
    3. 项目资料保密
    4. idea2app 工程师每日审核 AI 评估
    5. 合同签订并支付定金后，项目开发由 idea2app 公司分配的专业工程师完成